docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner works
+
+## Contrastive metric learning
```

Comment:
What if users have their own labeled data? Will we still use contrastive metrics learning instead of supervised learning?

Reply:
by default it's supervised